packages/synapse-react-client/src/style/components/entity_finder/_entity-finder.scss

```diff
@@ -32,22 +31,24 @@ $-splitter-width: 12px;
   width: 100%;
 }
 
+.MuiDialog-root {
+  .EntityFinder {
+    &__MainPanel.MainPanelDualPane,
+    &__MainPanel.MainPanelSearch {
+      // Hack to make the dialog responsive to the width of the entity finder
+      // We may be able to remove this if we remove/replace the EntityFinder's `AutoSizer` component with something other
+      // than the 'react-base-table' implementation
+      width: 85vw;
+    }
+  }
+}
```

Comment:
Extract this style to only apply it in a dialog. Otherwise, we can make the width of the entity finder 100%